### PR TITLE
Added config setting: SqlSettings.ReplicaMonitorIntervalSeconds

### PR DESCRIPTION
**Pull request by cwarnermm**
Documentation for: https://github.com/mattermost/mattermost/pull/22888

### CODE REVIEW

File: source/configure/database-configuration-settings.rst [hunk 1]
```diff
@@ -408,3 +408,25 @@ Replica lag settings
...
+  :systemconsole: N/A
+  :configjson: .SqlSettings.ReplicaMonitorIntervalSeconds
+  :environment: MM_SQLSETTINGS_REPLICAMONITORINTERVALSECONDS
+
+  Specifies how frequently unhealthy replicas will be monitored for liveness check. Mattermost will dynamically choose a replica if it's alive.
+
+Replica monitor interval (seconds)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. include:: ../_static/badges/allplans-selfhosted.rst
```
Comment (cwarnermm) [inline source/configure/database-configuration-settings.rst:423]: @agnivade - Is this config setting available for all plans and both self-hosted and cloud deployments? (or is this intended as a paid feature via Professional or Enterprise)?

Comment (agnivade) [reply source/configure/database-configuration-settings.rst:423]: It cannot be modified in cloud. But for self-hosted, it's available in all editions.

Comment (cwarnermm) [reply source/configure/database-configuration-settings.rst:423]: Thanks, @agnivade! Is the default used in Cloud deployments?

Comment (agnivade) [reply source/configure/database-configuration-settings.rst:423]: I am not sure, you'll have to ask the SRE team on that.

Comment (cwarnermm) [reply source/configure/database-configuration-settings.rst:423]: @nickmisasi - Can you confirm whether Cloud deployments are using this config setting, and the value being used, if not default, please?

**Review by agnivade — APPROVED** [review]

**Comment by cwarnermm** [comment]
@gabrieljackson - Is this config setting used for Cloud deployments? If so, what's the value set for Cloud?

**Pull request merged**